ruby_library/entities/library.rb

```diff
@@ -0,0 +1,100 @@
+require_relative '../storage/storage'
```

Comment:
please, include it inside main file, not here

Reply:
ok